nextcloud.json

```diff
@@ -0,0 +1,619 @@
+                    "data_type": "string",
+                    "column_name": "Status",
+                    "column_order": 2,
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric"
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "get file info",
+            "identifier": "get_file_info",
+            "description": "Get file info",
+            "verbose": ".",
+            "type": "generic",
+            "read_only": false,
+            "parameters": {
+                "path": {
+                    "description": "File path to get info",
+                    "data_type": "string",
+                    "required": true,
+                    "order": 0,
+                    "name": "path",
+                    "id": 1,
+                    "param_name": "path"
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.parameter.path",
+                    "data_type": "string",
+                    "column_name": "Path",
+                    "column_order": 0
+                },
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "column_name": "Status",
+                    "column_order": 1,
+                    "example_values": [
+                        "success",
+                        "failed"
+                    ]
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.summary",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric"
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        }
+    ],
+    "pip3_dependencies": {
+        "pypi": [
```

Comment:
we should instead specify this dependency in requirements.txt so it's included as a wheel along with it's subdependencies. that will pin the version and reduce the likelihood of unexpected dependency conflicts in the future